src/engraving/dom/slur.cpp

```diff
@@ -83,7 +83,7 @@ static ChordRest* searchCR(Segment* segment, track_idx_t startTrack, track_idx_t
 
 bool SlurSegment::isEditAllowed(EditData& ed) const
 {
-    if (ed.key == Key_Home && !ed.modifiers) {
+    if (ed.key == Key_Home && !ed.modifiers && ed.hasCurrentGrip()) {
```

Comment:
fixes the "reported" problem, that Home key can effectively be swallowed by a selected slur, even if no grips selected.